docs/guides/directive-breakdown/readme.md

```diff
@@ -0,0 +1,151 @@
+---
+authors: Matt Richerson <matthew.richerson@hpe.com>
+categories: provisioning
+---
+
+# Directive Breakdown
+
+## Background
+
+The `#DW` directives in a job script are not intended to be interpreted by the workload manager. The workload manager passes the `#DW` directives to the NNF software through the DWS `workflow` resource, and the NNF software determines what resources are needed to satisfy the directives. The NNF software communicates this information back to the workload manager through the DWS `DirectiveBreakdown` resource. This document describes how to interpret the information in the `DirectiveBreakdown`.
```

Comment:
"This document describes how to interpret..." should that be "how the WLM should interpret" ?